pairviewer.js

```diff
@@ -12,7 +12,17 @@ let currentRepoFilter = [];
 let currentPointFilter = [];
 let currentDirFilter = [];
 
+let MARKER_REFX = "2";
+let MARKER_REFX2 = "2.5";
```

Comment:
Two things:
1) Could you rather rename `MARKER_REFX`, `MARKER_REFX2`, and `MARKER_REFY` to `MARKER_REF_X`, `MARKER_REF_X2`, and `MARKER_REF_Y` respectively for readability?
2) The name `MARKER_REFX2` is a bit ambiguous. Could you maybe rename it (and `MARKER_REFX`) to something more descriptive?

Reply:
Okay. :)